easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -134,6 +134,71 @@ def cache_aware_func(toolchain):
 
     return cache_aware_func
 
+def get_subtoolchain_in_deps(parsed_ec, subtoolchain_name, current_tc_name):
```

Comment:
@bartoldeman This is begging for a dedicated unit test, to verify that it works as intended (and doesn't get broken in the future somehow)...